CHANGELOG.md

```diff
@@ -487,6 +487,7 @@
 - Work out a Budget's period based on the financial year
 - Users can see Current activities and Historic activities in different tabs
 - Reorder reports in a more intuitive manner
+<<<<<<< HEAD
```

Comment:
This probably shouldn't be here :)

Reply:
Oops! Completely missed that one. Thanks!!